src/App.tsx

```diff
@@ -1,14 +1,80 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodosFilters } from './types/TodosFilters';
+import { Filter } from './types/Filter';
+
+function getFilteredTodos(todos: Todo[], filter: Filter): Todo[] {
+  let filteredTodos = todos.filter(todo => {
+    switch (filter.option) {
+      case TodosFilters.Active:
+        return !todo.completed;
+      case TodosFilters.Completed:
+        return todo.completed;
+      case TodosFilters.All:
+      default:
+        return todo;
+    }
+  });
+
+  if (filter.query) {
+    filteredTodos = filteredTodos
+      .filter(({ title }) => title.toLowerCase()
+        .includes(filter.query.toLowerCase()));
+  }
+
+  return filteredTodos;
+}
+
+const initialState = {
+  todos: [],
+  loading: false,
+  selectedTodo: null,
+  filter: {
+    option: TodosFilters.All,
+    query: '',
+  },
+};
```

Comment:
🙃 I mean create constant only for this object `{
    option: TodosFilters.All,
    query: '',
  }`

Reply:
All other default values should be written as it was before

Reply:
> All other default values should be written as it was before

Yeah, it's unnecessary, update it